src/loaders/eventLoader.js

```diff
@@ -74,14 +74,13 @@ const loadEventLayer = async (config) => {
         startDate,
         styleDataItem,
         areaRadius,
-        showDataTable,
     } = config
 
     const period = getPeriodFromFilters(filters)
     const dataFilters = getFiltersFromColumns(columns)
     const d2 = await getD2()
 
-    config.isExtended = showDataTable
+    config.isExtended = loadExtended
```

Comment:
I've looked at this file in full, and see that object mutations like above fit with what is already there. I also see we're calling `delete` to remove a property. So probably it'd be good to refactor this file so that we are not mutating an existing object, but creating a new one (immutability). But this is out-of-scope for now, I think it's fine to keep things like this now, for consistency's sake.

Reply:
I think in the case of this file, it makes sense to leave things as they are. It's a big file and this PR only changes a few lines